src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -67,18 +67,15 @@ public void storeMessage(Message message) {
    * Either stores the user in datastore, or updates the messagesSent
    * property when a message is being stored.
    */
-  public void storeUser(String username, long messagesSent) {
+  public void storeUser(String username, long updateMessagesSent) {
```

Comment:
I'm not sure if you'll need to store the number of sent messages as a long -- I think a double or even an integer may be sufficient? I may be misunderstanding some part incorrectly, but this is just what I pulled from the variable naming!

Reply:
Yeah int should be fine.